RGPosadas/LitterRally
# US-3.4: DMs Page

Closes #227

**Pre-requisites**
Ensure that you have friends that you can chat with.

Steps to test: 
1. Run `mull-ui` and `mull-api`
2. Go to the messages page
3. Click on the person you wish to chat with
4. You should be able to chat just like in event messages.
